pkg/reconcilermanager/constants.go

```diff
@@ -141,14 +141,13 @@ const (
 	//HelmDeployNamespace is the OS env variable key for the Helm deploy namespace.
 	HelmDeployNamespace = "HELM_DEPLOY_NAMESPACE"
 
-	// HelmValues is the OS env variable key for the Helm chart values.
-	HelmValues = "HELM_VALUES"
+	// HelmValuesInline is the OS env variable key for the inline Helm chart values, formatted as a yaml
+	// string in the same format as the default values.yaml accompanying the chart.
+	HelmValuesInline = "HELM_VALUES_INLINE"
```

Comment:
`HelmValuesYAML = "HELM_VALUES_YAML"` might be more intuitive. as in, not a file path but the raw yaml. "inline" is more of an adjective than a noun.

Reply:
done